### PR TITLE
[4.0] Fix for 27192

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #27192 .

### Summary of Changes
More options added to cropperjs

### Testing Instructions
Read the detailed explanation in the issue

### Documentation Changes Required
NO, bug fix

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ded63dfff3e553a7c70e5c879395090d9c5e8947<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27255">issues.joomla.org/tracker/joomla-cms/27255</a>.</sub>

**Comment by brianteeman** [comment]
Thanks Dmitiris - I assumed it would use the plugin and not directly or I would have worked it out. doh<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27255">issues.joomla.org/tracker/joomla-cms/27255</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ded63dfff3e553a7c70e5c879395090d9c5e8947<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27255">issues.joomla.org/tracker/joomla-cms/27255</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27255">issues.joomla.org/tracker/joomla-cms/27255</a>.</sub>

File: administrator/components/com_templates/tmpl/template/default.php [hunk 1]
```diff
@@ -158,7 +158,7 @@
 			</form>
 		<?php elseif ($this->type == 'image') : ?>
 			<legend><?php echo $this->escape(basename($this->image['address'])); ?></legend>
-			<img id="image-crop" src="<?php echo $this->image['address'] . '?' . time(); ?>">
+			<img id="image-crop" src="<?php echo $this->image['address'] . '?' . time(); ?>" styles="max-width: 100%">
```
Comment (SharkyKZ) [inline administrator/components/com_templates/tmpl/template/default.php]: `styles` should be `style` or not?

Comment (dgrammatiko) [reply administrator/components/com_templates/tmpl/template/default.php]: Good catch

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**